### PR TITLE
Update 0-introduction.md

### PR DESCRIPTION
**Pull request by Abhishekmamidi123**

### CODE REVIEW

**Comment by duttakapil** [comment]
@djsbalakrishnan can you review this?

File: programming-languages/Python/2-curriculum/2-installing-python/0-introduction.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 <h3 align="center"> Installing python </h3>
 Topics Covered - Installing python in windows, Mac and Linux
 Note - Python3 is recommended
+
+
+If you want to use a different version, otherthan the recommended version Python3, then it is suggested to use Anaconda or miniconda. You can create your own environments accordingly based on your application.
```
Comment (djsbalakrishnan) [inline programming-languages/Python/2-curriculum/2-installing-python/0-introduction.md]: @Abhishekmamidi123 please correct the typo in otherthan to other than. But still note that all of the suggestions that I have made are on python3.

**Review by djsbalakrishnan — COMMENTED** [review]
@duttakapil yeah will review this.

File: programming-languages/Python/2-curriculum/2-installing-python/0-introduction.md (hunk 1 above)
Comment (Abhishekmamidi123) [reply programming-languages/Python/2-curriculum/2-installing-python/0-introduction.md]: It's better to mention the learners to use Anaconda or miniconda even if Python3 is recommended

**Comment by duttakapil** [comment]
This PR has way too many changes in it, mostly changes made by others that have already been merged. I'm struggling to understand what does this one exactly add that doesn't exist already?

**Pull request closed without merging**